powerauth-push-server/src/main/java/io/getlime/push/controller/rest/PushCampaignController.java

```diff
@@ -118,6 +121,7 @@ public PushCampaignController(PushCampaignRepository pushCampaignRepository,
      */
     @RequestMapping(value = "list", method = RequestMethod.GET)
     @ResponseBody
+    @ApiOperation(value = "Return a detailed list of campaigns", notes = "Restricted with all param")
```

Comment:
The message "Restricted with all param" does not say anything. Please provide a descriptive message, or remove "notes".